### ISSUE TITLE
Undeclared dependency on ERB

### ISSUE THREAD

**Issue by cluesque** [jnunemaker/httparty#398]
HTTParty has recently started depending on ERB, breaking scripts:

With `ruby 2.1.5p273`:

```
$ irb
irb(main):001:0> require 'rubygems'
=> false
irb(main):002:0> require 'httparty'
=> true
irb(main):003:0> HTTParty.get('https://api.github.com/users/jnunemaker/repos', query: {type: 'owner'}).code
NameError: uninitialized constant HTTParty::HashConversions::ERB
	from /Users/bill/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty/hash_conversions.rb:33:in `normalize_param'
	from /Users/bill/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty/hash_conversions.rb:15:in `block in to_params'
	from /Users/bill/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty/hash_conversions.rb:15:in `each'
	from /Users/bill/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty/hash_conversions.rb:15:in `map'
	from /Users/bill/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty/hash_conversions.rb:15:in `to_params'
	from /Users/bill/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty/request.rb:148:in `normalize_query'
	from /Users/bill/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty/request.rb:180:in `query_string'
	from /Users/bill/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty/request.rb:70:in `uri'
	from /Users/bill/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty/request.rb:157:in `setup_raw_request'
	from /Users/bill/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty/request.rb:98:in `perform'
	from /Users/bill/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty.rb:521:in `perform_request'
	from /Users/bill/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty.rb:459:in `get'
	from /Users/bill/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/httparty-0.13.4/lib/httparty.rb:559:in `get'
	from (irb):3
	from /Users/bill/.rbenv/versions/2.1.5/bin/irb:11:in `<main>'
irb(main):004:0> require 'erb'
=> true
irb(main):005:0> HTTParty.get('https://api.github.com/users/jnunemaker/repos', query: {type: 'owner'}).code
=> 200
```

This was introduced with [23b625e76c], but the dependency hasn't been added to the gemspec.  (Is that the correct fix?)

**Comment by oversee-ffleming** [comment]
I also ran into this today, opened [PR 399](https://github.com/jnunemaker/httparty/pull/399).

**Comment by jnunemaker** [comment]
Merged #399
